client/src/components/ChatDrawer/ChatDrawer.tsx

```diff
@@ -16,15 +16,17 @@ const ChatDrawer = ({ convos }: Props): JSX.Element => {
   return (
     <div>
       <Fragment>
-        <Button
-          onClick={(): void => setExpanded(!expanded)}
-          color="primary"
-          size="large"
-          variant="contained"
-          className={classes.expandBtn}
-        >
-          All Conversations
-        </Button>
+        <div className={classes.btnContainer}>
```

Comment:
You can use an MUI Box component here, it has flexbox built into it as well :)

Reply:
This is fixed.